__tests__/endpoint.test.js

```diff
@@ -81,14 +81,67 @@ describe("/api/articles/", () => {
       });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("The endpoint will respond with an array of comments from the given article ID", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
```

Comment:
We should check that the article_id is the correct value here based on the request

Reply:
We still need to address this please :) article_id should be 1, not typeof "number" 👍